### DISCUSSION TITLE
[Issue] Uploading large JMeter JTL File Cause to Throw an error of Request Entity Too Large

### DISCUSSION THREAD

**Discussion by Ntm2024** [ludeknovy/jtl-reporter#285] [General]
**Operation:** Upload large JMeter JTL file 
**File Size:** 4 GB
**Endpoint**: /items
**App Version**: tag: v4.10.2
**Environment Details:** 
- Ubuntu 22
- 64 bit OS
- Processor 13th Gen Intel® Core™ i7-1365U × 12

**Error screenshot:**
![jtl_error_large_file](https://github.com/ludeknovy/jtl-reporter/assets/171353045/f872ff00-2f59-4bc0-8ebe-0b438b746105)

**Comment by ludeknovy** [comment]
Hi @Ntm2024 , currently, there's a limit of 2GB.

**Reply by Ntm2024** [reply]
@ludeknovy Thanks for the quick reply. It would be great if you can share me some information on why the file limit is set to 2GB. 
Is there any option to change it by setting any configuration as mentioned in the doc(https://jtlreporter.site/docs/guides/large-file).

**Reply by ludeknovy** [reply]
Unfortunately there's no configuration for it.
Honestly, I dont remember why I set it to 2gb. Most likely it was due to some memory issue.
I can increase it, but you will need enough memory(it will be released after the processing) to process is due to issue described here: https://github.com/ludeknovy/jtl-reporter/issues/259, there's a proper fix but it has not been released yet.

**Reply by ludeknovy** [reply]
I've increased the limit to 5GB, you can give it try using the `fe` and `be` images with `latest` tag.

**Reply by Ntm2024** [reply]
@ludeknovy sorry I missed your reply. I have tried with latest image, but still the issue exist.

**Reply by ludeknovy** [reply]
What kind of error did you get. I am almost sure I've updated it everywhere where needed.
